Dockerfile

```diff
@@ -783,16 +783,16 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
-FROM sdk-go-1.23 AS sdk-govc
+FROM sdk-go-1.24 AS sdk-govc
 
 USER root
 RUN \
   mkdir -p /usr/libexec/tools /usr/share/licenses/govmomi && \
   chown -R builder:builder /usr/libexec/tools /usr/share/licenses/govmomi
 
-ENV GOVMOMIVER="0.30.2"
-ENV GOVMOMISHORTCOMMIT="9078b0b"
-ENV GOVMOMIDATE="2023-02-01T04:38:23Z"
+ENV GOVMOMIVER="0.46.3"
```

Comment:
There are a few breaking changes introduced in **0.47.0**. Given we're already jumping quite far from **0.30.2**, I thought it wise to start with **0.46.3** and then jump to >= **0.50.0** later.